Src/AutoFixture.xUnit.net.UnitTest/DelegatingFixture.cs

```diff
@@ -101,5 +102,7 @@ public object Create(object request, ISpecimenContext context)
         }
 
         internal Func<object, ISpecimenContext, object> OnCreate { get; set; }
+
+        internal Action<ICustomization> OnCustomize { get; set; }

```

Comment:
I might be missing something, but I'm not sure I understand how `Func` can be used here. I need to track that `sut.GetData()` calls the `fixture.Customize()` method twice with proper customizations in proper order. I store the actual *calls* in the `customizationLog` variable.

If I try to use a `Func`, I end up with smth like this:

```csharp
var customizationLog = new List<ICustomization>();
var fixture = new DelegatingFixture();
fixture.OnCustomize = c =>
{
  customizationLog.Add(c);
  return fixture;
};

var sut = new AutoDataAttribute(fixture);
// Exercise system
sut.GetData(method, parameterTypes);
// Verify outcome
Assert.False(customizationLog[0] is FreezeOnMatchCustomization);
Assert.True(customizationLog[1] is FreezeOnMatchCustomization);
// Teardown
```

Actually I still use it as an `Action` but now it takes even more space... Any suggestions how to simplify it? Thanks!